source/administration/object-management/object-lifecycle-management.rst

```diff
@@ -10,41 +10,46 @@ Object Lifecycle Management
    :local:
    :depth: 1
 
-MinIO Object Lifecycle Management allows creating rules for time or date
-based automatic transition or expiry of objects. For object transition,
-MinIO automatically moves the object to a configured remote storage
-tier. For object expiry, MinIO automatically deletes the object.
+MinIO Object Lifecycle Management allows creating rules for time or date based automatic transition or expiry of objects. 
```

Comment:
How about
> Use MinIO's Object Lifecycle Management to create rules ....

Reply:
"allows creating" is just odd as it.
It needs a `for` or `to create` instead. But I suggest the rewrite above.